.github/CODEOWNERS

```diff
@@ -349,9 +349,9 @@ packages/react-components/react-calendar-compat/stories @microsoft/cxe-prg
 packages/react-components/react-infolabel @microsoft/cxe-prg
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33378&commitId=c97b7ed895c2c213b3a4a3bc27e44899c366344f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|6| Changed|
|Avatar Converged.badgeMask.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1139| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2559| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>